### PR TITLE
Design Feedback: Dashboard

### PR DESCRIPTION
**Pull request by buberdds**
- fix random dashboard CSS issues (more charts issues will be address in another PR)
- prepare bar chart for incoming active accounts endpoint

screenshot with mocked Active Accounts data
```
[
    {
      active_accounts: 634,
      bucket_start: '2022-04-11T22:15:00Z',
    },
    {
      active_accounts: 438,
      bucket_start: '2022-04-11T22:10:00Z',
    },
...
]
```

Screenshot 2023-02-28 at 16 18 12

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -14,7 +14,13 @@ import './styles/index.css'
 import './locales/i18n'
 
 Axios.defaults.baseURL = process.env.REACT_APP_API
-const queryClient = new QueryClient({})
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
```
Comment (lubej) [inline src/index.tsx:20]: Does this affect rose price chart?

File: src/app/components/Select/index.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const StyledButton = styled(Button)(({ theme }) => ({
   height: '36px',
   minWidth: '135px',
   padding: `0 ${theme.spacing(4)}`,
-  borderRadius: '12px',
+  borderRadius: '36px',
```
Comment (lubej) [inline src/app/components/Select/index.tsx:23]: image

So the reason why I didn't set the `border-radius` to the "correct" value according to the designs, is that the dropdown looks a bit off. Not having the same `border-radius` as the trigger button.

File: src/app/components/charts/BarChart.tsx [hunk 3]
```diff
@@ -11,29 +11,58 @@ import { TooltipContent, type Formatters } from './Tooltip'
...
+const BarChartCmp = <T extends object>({
+  cartesianGrid,
+  data,
+  dataKey,
+  formatters,
+  rounded,
+  withLabels,
+  withBarBackground,
+}: BarChartProps<T>) => (
+  <ResponsiveContainer width="100%" height={80}>
```
Comment (lubej) [inline src/app/components/charts/BarChart.tsx:33]: Hardcoding the height here messes up `Transactions per day` chart

File: src/index.tsx (hunk 1 above)
Comment (buberdds) [reply src/index.tsx:20]: It's intentional global change, affects all queries.

File: src/app/components/Select/index.tsx (hunk 2 above)
Comment (buberdds) [reply src/app/components/Select/index.tsx:23]: dropdown list item hover state ? I will wait for Don's feedback in this case. thx for info

Comment (buberdds) [reply src/app/components/Select/index.tsx:23]: Don wants to keep border-radius from designs.

Comment (lubej) [reply src/app/components/Select/index.tsx:23]: Right, would be nice to sync the dropdown container as well - I didn't found any good way of doing this.

File: src/app/components/charts/BarChart.tsx (hunk 3 above)
Comment (buberdds) [reply src/app/components/charts/BarChart.tsx]: reverted as this is handled in one of your PR

**Review by lubej — APPROVED** [review]

**Pull request merged**